terraform.tfvars

```diff
@@ -17,3 +19,9 @@ desired_capacity = 2
 instance_type = "t2.micro"
 
 ecs_aws_ami = "ami-c1a6bda2"
+
+owner = "alexander.vincent@ga.gov.au"
+
+ssh_ip_address = "192.104.44.129/32"
```

Comment:
Please remove this and inject it by setting the environment variable TF_VAR_ssh_ip_address.